tests/generic_config_updater/test_bgpl.py

```diff
@@ -14,6 +14,19 @@
 
 logger = logging.getLogger(__name__)
 
+def get_bgp_listener_config(duthost):
```

Comment:
>_config [](http://example.com/codeflow?start=20&length=7)

The naming is not proper. `show` command does not mean any configuration, even not running config. Suggest `show_bgp_monitor`